R/functions-Spectrum.R

```diff
@@ -882,7 +898,7 @@ descendPeak <- function(mz, intensity, peakIdx = NULL, signalPercentage = 33,
 #' of m/z and intensity pairs than the spectrum with index `main` in `x`, also
 #' all other related information is taken from this spectrum.
 #'
-#' @author Johannes Rainer
+#' @author Johannes Rainer, Sigurdur Smarason
```

Comment:
I Sigurdur Smarason @SiggiSmara has the co-author of the new code, could you also add hime as a contributor (`role = "ctb"`) to the `DESCRIPTION` file, please.

Reply:
OK

Reply:
:smiley: thanks! @lgatto